src/components/UI/JSONSchemaForm/test.schema.json

```
@@ -0,0 +1,190 @@
+                    "title": "Number with min/max values",
+                    "type": "number"
+                }
+            },
+            "title": "Numeric fields",
+            "type": "object"
+        },
+        "objectFields": {
+            "properties": {
+                "object": {
+                    "properties": {
+                        "age": {
+                            "type": "integer"
+                        },
+                        "name": {
+                            "type": "string"
+                        }
+                    },
+                    "title": "Object",
+                    "type": "object"
+                },
+                "objectRequired": {
+                    "description": "Object field with required properties. Submit the form to see validation errors.",
+                    "properties": {
+                        "age": {
+                            "type": "number"
+                        },
+                        "name": {
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "name"
+                    ],
+                    "title": "Object with required",
+                    "type": "object"
+                }
+            },
+            "title": "Object fields",
+            "type": "object"
+        },
+        "stringFields": {
+            "properties": {
+                "string": {
+                    "description": "String field with title and description.",
+                    "title": "String",
+                    "type": "string"
+                },
+                "stringCustomLabels": {
+                    "description": "String field with enumerated values and custom labels.",
+                    "oneOf": [
+                        {
+                            "const": "firstValue",
+                            "title": "First Value"
+                        },
+                        {
+                            "const": "secondValue",
+                            "title": "Second Value"
+                        },
+                        {
+                            "const": "thirdValue",
+                            "title": "Third Value"
+                        }
+                    ],
+                    "title": "String enum with custom labels",
+                    "type": "string"
+                },
+                "stringEnum": {
+                    "description": "String field with enumerated values.",
+                    "enum": [
+                        "firstValue",
+                        "secondValue",
+                        "thirdValue"
+                    ],
+                    "title": "String enum",
+                    "type": "string"
+                },
+                "stringFormat": {
+                    "description": "String field with specific format (email, uri, date, ipv4, etc.).",
```

To make it clear which format is valid here:

```suggestion
                    "description": "String field with format 'ipv4'.",
```